cmd/kindnetd/cni.go

```diff
@@ -114,8 +114,8 @@ func (c *CNIConfigWriter) Write(inputs CNIConfigInputs) error {
 		os.Remove(f.Name())
 		return err
 	}
-	f.Sync()
-	f.Close()
+	_ = f.Sync()
+	_ = f.Close()
```

Comment:
on the fence about these (look at the context), but at least `_ =` is trivial to grep later. error overhaul comes after we revive the log shim work post 0.5 ...